mockk/common/src/test/kotlin/io/mockk/it/ValueClassTest.kt

```diff
@@ -0,0 +1,56 @@
+package io.mockk.it
+
+import io.mockk.*
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class ValueClassTest {
+
+    private val mock = mockk<DummyService>()
+
+    @Test
+    fun `value class object as return value`() {
```

Comment:
Can you please use single word test names, e.g. valueClassObjectAsReturnValue? Turns out android does not really like backtick-named tests with more than one word :(

Reply:
Sorry i missed this earlier, btw :(

Reply:
sure, no problem.